### ISSUE TITLE
Alt text should be descriptive

### ISSUE THREAD

**Issue by discospiff** [nityak04/IS7024#8]
Put a description of the image in the alt attribute, so that users who cannot see the image will understand what it represents.

https://github.com/nityak04/IS7024/blob/1f757498065ae1aef122b79a53c3703a8e96c36c/TheModernIlluminati/TheModernIlluminati/Pages/AboutUs.cshtml#L85
